saleor/graphql/account/mutations/account.py

```diff
@@ -223,7 +224,7 @@ def perform_mutation(cls, _root, info, **data):
         return cls.success_response(user)
 
 
-class AccountAddressCreate(ModelMutation):
+class AccountAddressCreate(ModelMutation, I18nMixin):
```

Comment:
I know the code of `I18nMixin` is not yours but the name of that mixin...
What concerns me is that I would expect it does some translation underneath but what it actually does is only a validation of address field while be named `I18nMixin`.

Reply:
Yes we probably should rename it. Maybe `I18nAddressMixin`
@maarcingebala what you think?